.github/workflows/hybrid-integration-test.yaml

```diff
@@ -0,0 +1,68 @@
+name: integration test
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
```

Comment:
mchurichi said:

can we use commit hashes instead of tags in all steps? it's a good practice in terms of supply chain security